ptu/global.ini

```diff
@@ -41934,17 +41934,17 @@ blackboxillegal_obj_marker_03=Lieferort
 blackboxillegal_obj_short_01=Gehe zur Position des Wracks
 blackboxillegal_obj_short_02=Schnapp dir die Black Box
 blackboxillegal_obj_short_03=Liefere die Black Box ab
-blackboxillegal_timed_0001=Es heißt, dass es legale Typen gibt, die auf Sie zukommen, also müssen Sie schnell handeln. 
-blackboxillegal_timed_0002=Sie fangen bereits an, etwas hitzig zu werden, also musst du schnell handeln. 
+blackboxillegal_timed_0001= Man munkelt, dass einige Juristen im Anmarsch sind, also müssen du dich beeilen.  
+blackboxillegal_timed_0002=Sie sind bereits in der Kritik, also musst du schnell handeln.  
 blackboxillegal_timed_0003=Oh, und je früher, desto besser, okay? 
 blackboxillegal_timed_0004=Das muss schnell erledigt werden, okay? 
 blackboxillegal_title=~mission(Title)
 blackboxillegal_title_0001=Black Box Recovery
 blackboxillegal_title_0002=~mission(Item) snag
 blackboxillegal_title_0003=Erase a Problem
 blackboxillegal_title_0004=Bye Bye Black Box
-blackboxlegal_danger_0001=Was auch immer mit der ~mission(Ship) passiert ist, könnte immer noch in der Gegend lauern. Seien Sie vorsichtig. 
-blackboxlegal_danger_0002=Wir wissen nicht, was oder wer diesen Vorfall verursacht hat, also halten Sie die Augen offen. 
+blackboxlegal_danger_0001=Was auch immer mit der ~mission(Ship) passiert ist, könnte sich immer noch in der Gegend befinden. Sei vorsichtig. 
```

Comment:
`Was auch immer bei der ~mission(Ship) passiert ist,`

Reply:
wie wäre es mit: 
Was auch immer der ~mission(Ship) zugestoßen ist, es könnte sich immer noch in der Gegend befinden. Sei vorsichtig.

Reply:
Joar